for/wordpress.md

```diff
@@ -182,7 +182,7 @@ Codeception allows to combine tests written in different formats. If are about t
 There is no standard Gherkin steps built in. By writing your feature files you can get code snippets which should be added to `AcceptanceTester` class. 
 
 ```
-composer exec codecept gherkin:snippets
+./vendor/bin/codecept exec codecept gherkin:snippets
```

Comment:
This example used `composer exec` for some reason.
Please remove `exec codecept` to make updated code valid.

Reply:
Apologies, this was a bad copy/paste

To confirm, original code snippet was 

`composer exec codecept gherkin:snippets`

So it should be updated to 

`./vendor/bin/composer exec codecept gherkin:snippets`

correct?

Reply:
Actually never mind, it should go back to just being `composer`, as installing the package requires composer to be installed globally anyway.

Reply:
Sorry, it's after work hours and I've had some wine :-)

If I'm understanding this right, running `composer exec codecept` is the same as running ./vendor/bin/codecept`, which is why you've suggested removing `exec codecept`, correct?

In that case, does it make sense to update all examples to `composer exec codecept` and not `./vendor/bin/codecept`, because installing Codeception requires composer to be installed globally anyway?

Reply:
Updated to `./vendor/bin/codecept`